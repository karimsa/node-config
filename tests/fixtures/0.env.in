__CONFIG_ENCRYPTION_KEY__=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA9jDm71rES43UFLBqNHXtDV1h0aA1pYB5W9vWRXVWw6xQ+6M44dNYo29Dv2jqv+EQD4hMnPLpEs7keA4ZSjxEG05LyW4OOqtrtQtKH7BtsDvvQoqHxWgZSVBx3FZrODziYm/xNpeae3+SHxjiV0mi6QgT6AoYW4uPrIKOKPmb1zxR0qT2xDnndnR4XsCn6gf319pFnAnfuOIuG8Umr2ZDWQQhG/ybc6Q6XFMULrW6lQAKIC+5jewNI/x7HRbI9bfaXu+biNCfrTEpxM2gHfw+2LCjYhf6AA03fJJOe96HCfFl00f3Hl6GU0pqGJkFk0xfC5+0RS25NbLQGoK3Icx+bQIDAQAB

SOME_ID=abc
SOME_KEY_SECRET=def