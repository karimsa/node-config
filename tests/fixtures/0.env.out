__CONFIG_ENCRYPTION_KEY__=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA9jDm71rES43UFLBqNHXtDV1h0aA1pYB5W9vWRXVWw6xQ+6M44dNYo29Dv2jqv+EQD4hMnPLpEs7keA4ZSjxEG05LyW4OOqtrtQtKH7BtsDvvQoqHxWgZSVBx3FZrODziYm/xNpeae3+SHxjiV0mi6QgT6AoYW4uPrIKOKPmb1zxR0qT2xDnndnR4XsCn6gf319pFnAnfuOIuG8Umr2ZDWQQhG/ybc6Q6XFMULrW6lQAKIC+5jewNI/x7HRbI9bfaXu+biNCfrTEpxM2gHfw+2LCjYhf6AA03fJJOe96HCfFl00f3Hl6GU0pqGJkFk0xfC5+0RS25NbLQGoK3Icx+bQIDAQAB

SOME_ID=abc
SOME_KEY_SECRET=$$[ASZpK28kb/0FBcC+oJTQdUm4NoMiw/tCgrrMPyRmWR7TieX/VnafrpRHK3n1nHKVi4UMqUCHzGLts0BC9A4melWkrSU0KtBfa9iW9NwOECuND/YAk+vpAVNXb41bKHngnbJCVJz7tM+nMK4nzeL6LDfU/2W8M7ARLmi7IMh1W6kXh/854SD9QQN7mvMYZtHNDKrlmN7UCmDw/12LDQmVLoKYvN2gwlTSJ9RYA+AMLJg/8hlzWNx/f2WgR+PaBfN7DVmk+1lLrV59j9CBBxvP7/a1hvVpRxHQXPDPU0hc4JHyv3XIsfQksW39O09NQPGtmnMQsxHVLYTXNTdC12Qx2NloHTsvAW2xuoeQNtdvr7JtqMNXOpMjdTFUc5ZoEEhJuw]